Orange/canvas/document/editlinksdialog.py

```diff
@@ -278,11 +277,13 @@ def mouseMoveEvent(self, event):
                     (downPos - event.pos()).manhattanLength() > \
                         QApplication.instance().startDragDistance():
                 # Start a line drag
-                line = QGraphicsLineItem(self)
+                line = LinkLineItem(self)
                 start = self.__dragStartItem.boundingRect().center()
                 start = self.mapFromItem(self.__dragStartItem, start)
-                line.setLine(start.x(), start.y(),
-                             event.pos().x(), event.pos().y())
+
+                eventPos = event.pos()
+                # False positive; pylint: disable=too-many-function-args
```

Comment:
@janezd 

This obstructs the readability/flow/interpretation of the code. When I read this I expect the comment to elucidate the following code. It is only at the middle that I realize I am on the wrong path and need to backtrack.

Put `# pylint: ...` at the beginning so the reader can bail out earlier. Although I would prefer to remove it altogether.